docs/src/documentation/05-development/04-migration-guides/01-v15.mdx

```diff
@@ -10,10 +10,41 @@ redirect_from:
 This migration guide focuses on the process of migrating from Orbit v14 to v15.0, as some breaking changes were introduced.
 With this guide, we aim to walk through all the breaking changes and how they can be addressed, allowing the migration to be smoother and effortlessly.
 
+This version marks the removal of `styled-components`. The whole styling of Orbit components is now done using Tailwind exclusively. This also means that our support for `styled-components` related issues is now finished.
+
 ## Breaking changes
 
+### OrbitProvider changes
+
+The `OrbitProvider` component has changed and no longer includes the `ThemeProvider` from `styled-components`.
+
+**This should not have any impact if you are not using `styled-components` in your project.**
+
+The `useTheme` hook is still available and can be used to access the theme values, as usual.
+
+If you are still using `styled-components` in your project, you will need to wrap your application (and the `OrbitProvider`) with the `ThemeProvider` from `styled-components` and pass it the same `theme` you are passing to `OrbitProvider`.
+
+```jsx
+import { ThemeProvider } from "styled-components";
+import { OrbitProvider } from "@kiwicom/orbit-components";
+
+const App = () => (
+  <ThemeProvider theme={theme}>
+    <OrbitProvider theme={theme}>
+      <App />
+    </OrbitProvider>
+  </ThemeProvider>
+);
+```
+
 ### mediaQueries util function removed
 
 The `mediaQueries` function is no longer available. This was a styled-components util function that is now no longer expected to be used, as we move away from StyledComponents.
 If needed, the tokens and breakpoint values are still accessible via the `getBreakpointWidth` function and the `QUERIES` and `TOKEN` constants.
 Check [the documentation](/development/utilities/media-queries/) for more information on how to use these.
+
+### rtl utils removed
+
+The `rtl` utility functions have been removed. They were used to handle RTL styles in the `styled-components` version of Orbit.
```

Comment:
Maybe add a comment as to how it works with Tailwind/why it's okay for this to be removed.

Also, if folks were relying on them, would it make sense to provide a code snippet replacement?

(same question for `mediaQueries` above actually?)

Reply:
I did some research and did not find relevant usages for those, hence why I didn't put that much effort into writing alternatives.

I did not write anything about how to achieve results in Tailwind because teams that are not yet in Tailwind and might need something will have to decide on how to handle these (I don't know if there is any, tbh).

I can add a few examples in the style of before and after scenarios, but I am afraid they will either be too incomplete (I believe it's _too much_ to cover all scenarios) or too specific, so I am not entirely sure about what to add.

Reply:
Okay, got it.

Maybe it still make sense to just point out to Tailwind docs for RTL and media queries
Maybe we could still offer a permalink to the previous RTL utility functions for folks to copy them over in their codebases in case they'd need them?